analyzer/src/main/kotlin/managers/Bundler.kt

```diff
@@ -141,7 +141,9 @@ class Bundler : PackageManager() {
         dependencyList.forEach {
             parseDependency(workingDir, it, packages, scopeDependencies, errors)
         }
-        scopes.add(Scope(groupName, true, scopeDependencies.toSortedSet()))
+
+        val delivered = !listOf("development", "test").contains(groupName.toLowerCase())
```

Comment:
Maybe nicer syntax:

    val delivered = groupName.toLowerCase() !in listOf("development", "test")